libs/designsystem/src/lib/components/button/button.component.ts

```diff
@@ -17,6 +17,9 @@ export enum ButtonSize {
   LG = 'lg',
 }
 
+const ATTENTION_LEVEL_4_DEPRECATION_WARNING =
+  'Deprecation warning: The "kirby-button" support for using input property "attentionLevel" with the value "4" will be removed in a future release of Kirby designsystem. Usage of attention-level 4 will forcibly be rendered as a attention-level 3 button.';
```

Comment:
```suggestion
  'Deprecation warning: The "kirby-button" support for using input property "attentionLevel" with the value "4" will be removed in a future release of Kirby designsystem. Usage of attention-level 4 is rendered as an attention-level 3 button.';
```

Reply:
Consider changing it to this, otherwise it might sound like it is in a future release that attention level 4 is mapped to attention level 3.

Reply:
-fixed